common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -47,6 +49,12 @@ public String getFieldName() {
     }
 
     public String getJsonName() {
+        if (parent instanceof Overlay) {
+            // "FeatureName.OverlayName.settingName"
+            return parent.getClass().getDeclaringClass().getSimpleName() + "."
+                    + parent.getClass().getSimpleName() + "." + field.getName();
+        }
+        // "FeatureName.settingName"
         return parent.getClass().getSimpleName() + "." + field.getName();
```

Comment:
Do we want to normalize the names to initial-lowercase? Like how we do for the translation keys. so like "fakeFeature.dummyOverlay.horizontalSize" etc.

In my eyes, that'd look better.